card/spec/models/card/create_spec.rb

```diff
@@ -31,7 +31,7 @@
 describe Card, "created by Card.create with valid attributes" do
   before(:each) do
     Card::Auth.as_bot do
-      @b = Card.create :name=>"New Card", :content=>"Great Content"
+      @b = Card.create name: "New Card", content: "Great Content"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.